src/main/java/io/github/debuggyteam/architecture_extensions/api/BlockType.java

```diff
@@ -88,7 +88,7 @@ public TypedGroupedBlock register(BlockGroup group, BlockGroup.GroupedBlock grou
 		var baseBlock = groupedBlock.baseBlock().get();
 
 		var blockSettings = QuiltBlockSettings.copyOf(baseBlock).strength(strength);
-		groupedBlock.mapColor().map(mapColor -> blockSettings.mapColor(mapColor));
+		groupedBlock.mapColor().map(mapColor -> blockSettings.mapColor(mapColor)); // replace lambda with method reference?
```

Comment:
either do it or don't. Don't leave the comment here, that's what IDE inspections are for.